content/tutorials/r_beginners/src/data_inlezen.Rmd

```diff
@@ -254,54 +254,53 @@ table(dfSurvey2$Sex) #controle of de geslachten juist ingevuld zijn
 
 #Googlesheets
 
-- Maakt gebruik van het **googlesheets** package
-- Je moet jezelf authoriseren via de functie `gs_auth()`
+- Maakt gebruik van het **googlesheets4** package
+- Je moet jezelf authoriseren via de functie `gs4_auth()`
     - dit moet slechts periodiek
     - je wordt verwezen naar een login website voor google
     - je geeft toegang aan tidyverse-googlesheets
         - ofwel krijg je dan een code die je in R moet plakken
         - ofwel zal je direct een melding krijgen dat de authorisatie gelukt is
```

Comment:
Zie echter https://googlesheets4.tidyverse.org/reference/gs4_auth.html#details. Dus deze sectie kan eventueel achterwege gelaten worden. Voor publieke sheets is authentificatie niet nodig. Enkel wanneer je wil schrijven naar een sheet of private sheets wil inlezen en in dat geval zal automatisch gevraagd worden voor toestemming.

Reply:
De meeste sheets zijn private die we inlezen, en ik vind het op zich niet verkeerd dat duidelijk is waar de authorisatie gebeurt, bv bovenaan een script, zodat de scripts niet plotseling vastlopen tijdens het runnen